modules/nf-core/fastme/main.nf

```diff
@@ -0,0 +1,53 @@
+process FASTME {
+    tag "$infile"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/fastme:2.1.6.1--hec16e2b_1':
+        'biocontainers/fastme:2.1.6.1--hec16e2b_1' }"
+
+    input:
+    path infile
+    path topo
```

Comment:
Thinks this is referring to an initial tree topology to use? Might be nice to give it a more descriptive name, something like `initial_tree` or the likes.